scripts/mkcloud

```diff
@@ -476,8 +476,7 @@ function setupadmin()
     wait_for 300 1 "ping -q -c 1 -w 1 $adminip >/dev/null" 'crowbar admin VM'
     wait_for_crowbar_ssh
 
-    echo "waiting some more for sshd+named to start"
-    sleep 25
+    wait_for 20 2 "nc -z $adminip 22 && nc -z $adminip" 'sshd+named to start'
```

Comment:
the comment was wrong in the first place, we never wait for named within this function
therefore dropping the named part